src/aarch64/ffi.c

```diff
@@ -316,74 +321,81 @@ extend_integer_type (void *source, int type)
     }
 }
 
+#if defined(_MSC_VER)
+void extend_hfa_type (void *dest, void *src, int h);
+#else
 static void
 extend_hfa_type (void *dest, void *src, int h)
 {
   ssize_t f = h - AARCH64_RET_S4;
   void *x0;
 
   asm volatile (
-	"adr	%0, 0f\n"
```

Comment:
It's hard to tell what's going on here. Did you make whitespace changes (tabs/spaces)?  Please try to remove the whitespace changes so we can see what's really going on.

Reply:
have incorporated your suggestion.